addons/mail/static/src/core/common/composer.xml

```diff
@@ -154,7 +154,7 @@
 </t>
 
 <t t-name="mail.Composer.quickActionButton">
-    <button class="btn border-0 rounded-circle" t-att-class="{ 'o-large': ui.isSmall and env.inChatWindow, 'o-small': !ui.isSmall or !env.inChatWindow }" t-attf-class="{{ action.btnClass }}" t-att-title="action.name" t-on-click="(ev) => action.onClick?.(ev)" t-att-data-hotkey="action.hotkey" t-att-disabled="action.disabledCondition or areAllActionsDisabled" t-att-name="action.id" t-ref="{{ action.id }}"><i class="fa-fw fa-lg" t-att-class="action.icon"/></button>
```

Comment:
`quickActionButton` also takes buttons on the right of composer, e.g. emoji button or send button. They must be disabled with this condition. Most likely should not remove this from the template.